SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestMultipleOwnership/SpatialTestMultipleOwnership.cpp

```diff
@@ -58,6 +58,17 @@ void ASpatialTestMultipleOwnership::PrepareTest()
 		RegisterAutoDestroyActor(MultipleOwnershipPawn1);
 		RegisterAutoDestroyActor(MultipleOwnershipPawn2);
 
+		for (ASpatialFunctionalTestFlowController* FlowController : GetFlowControllers())
+		{
+			if (FlowController->WorkerDefinition.Type == ESpatialFunctionalTestWorkerType::Client)
+			{
+				AController* Controller = Cast<AController>(FlowController->GetOwner());
```

Comment:
Assuming this should never ever never ever fail, can we make it a CastChecked please?

Reply:
I have opted for an assertIsValid below instead, which should serve a similar purpose, I think

Reply:
Can you do a if(!AssertIsValid) ? This could prevent a crash.

Reply:
ah yes, can do